org/cloudfoundry.yml

```diff
@@ -6165,7 +6165,7 @@ orgs:
         privacy: closed
         repos:
           cf-k8s-api: admin
-          cf-k8s-secrets: admin
+          korifi-secrets: admin
```

Comment:
This one was not renamed, since it is secret anyway and makes sense to be named after the working group